extension/src/debug.ts

```diff
@@ -13,62 +13,77 @@ let strNoADPerr: string = AutoLispExt.localize("autolispext.debug.nodap", "doesn
 let strNoACADerr: string = AutoLispExt.localize("autolispext.debug.noacad", "doesn’t exist. Verify and correct the folder path to the product executable.");
 let acadPid2Attach = -1;
 
-let attachCfgName = 'AutoLISP Debug: Attach';
-let attachCfgType = 'attachlisp';
-let attachCfgRequest = 'attach';
+const attachCfgName = 'AutoLISP Debug: Attach';
+const attachCfgType = 'attachlisp';
+const launchCfgName = 'AutoLISP Debug: Attach';
+const launchCfgType = 'launchlisp';
+const attachCfgRequest = 'attach';
```

Comment:
do you know why do we have this only for attach?

Reply:
I think maybe this is a technical debt from the beginning.